gql/README.md

```diff
@@ -189,6 +191,76 @@ void main() {
 }
 ```
 
+## `package:gql/schema.dart` (experimental)
+
+Higher-level schema type definitions derived from the `gql/ast.dart` asts.
+Adds convenience attributes and type dereferencing facilities.
+
+**NOTE**: does not currently have field resolution.
+
+```dart
+import "package:gql/language.dart" as lang;
```

Comment:
Please put these in `example/` as well. There are some `dartfmt` and `dartanalyzer` issues here.